home/ivp-planning/machines/selecting-and-moving-objects.md

```diff
@@ -14,11 +14,11 @@ To select an object in the [2D-](../user-interface/the-2d-panel.md) or [3D-panel
 It is also possible to select multiple objects by holding the **Ctrl** key while clicking on them one after another.
 {% endhint %}
 
-In the [2D-panel](../user-interface/the-2d-panel.md), you can identify selected objects by a blueish tint and a frame around the objects that displays their x- and y-size. Parts of this frame can also be used to [move](move-objects.md#moving-objects-in-the-2d-panel), [rotate ](scale-and-rotate-objects.md#rotating-objects-in-the-2d-panel)or [scale ](scale-objects.md#scaling-objects-in-the-2d-panel)objects in 2D.
+In the [2D-panel](../user-interface/the-2d-panel.md), you can identify selected objects by a blueish tint and a frame around the objects that displays their x- and y-size. Parts of this frame can also be used to [move](move-objects.md#moving-objects-in-the-2d-panel), [rotate ](rotate-objects.md#rotating-objects-in-the-2d-panel)or [scale ](scale-objects.md#scaling-objects-in-the-2d-panel)objects in 2D.
```

Comment:
Link on scale is broken

Reply:
removed link and information as i remember no way to scale in 2d oO